### ISSUE TITLE
Outcome

### ISSUE THREAD

**Issue by sadiekelly** [globaldothealth/outbreak-schema#2]
*Minimum EpiCore variable = Outcome of illness

The response options on the day0 schema are recovered | death | ongoing post-acute condition. 
The response options on the T0 toolkit form are Still sick | Cured | Sequelae | Defaulter | Death. 

It is unlikely that G.h data sources will differentiate between 'Still sick' and 'Sequelae' hence 'Ongoing post-acute condition' has been used.

Should response options be aligned as much as possible i.e. 'Recovered' --> 'Cured'

**Comment by abhidg** [comment]
For reference, T0 Toolkit: https://www.who.int/emergencies/outbreak-toolkit/standardized-data-collection-tools/t0-initial-case-investigation-form

**Comment by lauramerson** [comment]
recovered = cured
death = death
ongoing post-acute condition = sequelae
Still sick and Defaulter should be added to the day0 schema. 
Still sick is not the same as sequelae. Still sick is a part of an ongoing acute illness. Sequelae indicates the recovery from an acute illness, but with ongoing symptoms.

**Comment by sadiekelly** [comment]
thanks @lauramerson! Should the outcome options also be aligned with those on the ICC? The situation for Gh and ICC data differs though so not all outcome options are relevant or can be clearly aligned to both sets of data

**Comment by lauramerson** [comment]
Yes, good point. We should have all these options available on ICC. I'll address that.

**Comment by sadiekelly** [comment]
@aimeehan1 @JacqSauer @kelseytoups @julianasopko

**Comment by sadiekelly** [comment]
Recommendation: align outcome options in Day0 schema with those in the T0 toolkit Still sick | Cured | Sequelae | Defaulter | Death for consistency. Most likely reported responses are Death and Cured, other responses may be used only rarely.
